libopae/pluginmgr.c

```diff
@@ -71,6 +73,42 @@ STATIC opae_api_adapter_table *adapter_list = (void *)0;
 static pthread_mutex_t adapter_list_lock =
 	PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP;
 
+
+#define MAX_PLUGINS PLUGIN_SUPPORTED_DEVICES_MAX
+static plugin_cfg plugin_list[MAX_PLUGINS];
```

Comment:
I'm hesitant to set such limits..  If we only use a few of the entries, then space is wasted.  If we use a lot, we have to check for and handle the overflow case.
Can we use a linked list so that we allocate space as we go?

Reply:
can we do that in another PR?

Reply:
Yeah, sure.